conda/recipes/libraft/meta.yaml

```diff
@@ -66,9 +66,9 @@ outputs:
         - librmm
     requirements:
       run:
+        - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
```

Comment:
Since the `libraft-headers` package doesn't explicitly list `cudatoolkit` as a `build` or `host` dependency, I don't think `pin_compatible` will work as intended.

We can inspect the packages that result from the CI build to confirm.

There are docs on `pin_compatible` here: https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#extra-jinja2

To account for this limitation, you can use the `cuda_spec` variable that's at the top of the recipe if necessary.

Reply:
Yeah I noticed the same and was going to suggest just adding a host req but cuda_spec is cleaner.